composer.json

```diff
@@ -23,7 +23,8 @@
         "pear/versioncontrol_git": "@dev",
         "phing/phing": "^2.14",
         "phpmd/phpmd": "^2.4",
-        "phpspec/prophecy-phpunit": "^2"
+        "phpspec/prophecy-phpunit": "^2",
+        "typhonius/acquia-php-sdk-v2": "^2.0"
```

Comment:
@acidaniel I think Bec was interested in not adding this package to `the-build` -- I wonder if there's a way to only add it if a developer chooses "Acquia" as the hosting provider during setup?

I agree that it makes development on top of the API pretty seamless and I've benefited from it on DHS.

Reply:
Maybe adding it as a suggested package would work to begin with? https://getcomposer.org/doc/04-schema.md#suggest

```json
{
    "suggest": {
        "typhonius/acquia-php-sdk-v2": "Needed to use the phing AcquiaTask"
    }
}
```

There are apparently some paths forward for programmatically moving suggestions from suggested to required: https://stackoverflow.com/questions/27212257/is-there-a-way-to-interactively-install-suggested-composer-packages

Might be something to think about!

Reply:
@acidaniel One last follow-up based on our slack conversation:

> If we made it suggested, could we add a task that checks for the dependency and provides helpful output on failure? Then we could make any of the acquia tasks be dependent on that one.

Reply:
@jesconstantine I have added the `typhonius/acquia-php-sdk-v2` to the `suggest` key, and also I have updated the task `setup-host` that will include the package to the main `composer.lock` of the project and add the package only if `acquia` is selected as a hosting.

Reply:
Great! I just added a follow up question in a comment at that part of the code: https://github.com/palantirnet/the-build/pull/218#discussion_r1128195681

Reply:
@jesconstantine I have removed this from suggested and I have added this directly to a `composer require` IF the acquia provider is selected as a hosting vendor.